site/content/docs/0.0/utilities/z-index.md

```diff
@@ -15,11 +15,11 @@ We call these "low-level" `z-index` utilities because of their default values of
 {{< /callout >}}
 
 {{< example class="bd-example-zindex-levels position-relative" >}}
-<div class="z-3 position-absolute p-5"><span>z-3</span></div>
-<div class="z-2 position-absolute p-5"><span>z-2</span></div>
-<div class="z-1 position-absolute p-5"><span>z-1</span></div>
-<div class="z-0 position-absolute p-5"><span>z-0</span></div>
-<div class="z-n1 position-absolute p-5"><span>z-n1</span></div>
+<div class="z-3 position-absolute p-spacious"><span>z-3</span></div>
```

Comment:
Please don't take this comment for granted and don't modify the PR. This is a complex one to review, so I'll put some comments here and there with my 2 cents. Hopefully, in the end, with your rules, some comments, and the designers inputs, we might create a cool mapping table for usage, but also for migrations.

Here, the previous value is 60px. So I would rather use as a real project "huge" (56px) or "jumbo" (64px) to have the same rendering.
The previous rule is that the values doubled between **0,1, 2, 3, 4, 5 → 0, 5px, 10px, 20px, 30px, 60px**.
If we keep this rule I would rather use for equivalent usages and easier migration **none, shortest, shorter, medium, taller, jumbo** instead of the current **none, shortest, shorter, medium, tall, spacious**.

Reply:
The rule that was set as follows;
0: none,
1: shortest, (4px)
2: short, (12px)
3: tall, (24px)
4: tallest, (40px)
5: huge (56px)